tensorflow/python/ops/script_ops.py

```diff
@@ -370,6 +376,24 @@ def eagerly_executed_grad(*dy):
         is_grad_func=True)
 
 
+# NOTE(lithuak): this function as a layer of indirection was added with one
+# specific purpose: as a workaround for github issue #35084.
+# It does all the same as `eager_py_func` used to do with one difference:
+# it can be used to instruct underlying EagerFunc not to use `tape_cache`
+# to avoid memory leak. When the issue #35084 is fixed - this function should
+# be removed, its body should be moved back to become the body of
+# `eager_py_func` and all the call sites should be reverted to
+# using `eager_py_func` without `use_tape_cache` argument of any value.
+def _eager_py_func(func, inp, Tout, name=None, use_tape_cache=True):
```

Comment:
The lint checker is complaining that this function does not have a docstring. Can you please add a minimal one (which documents the arguments and the purpose of this function). In particular, the "NOTE" above can go be moved to the docstring.